source/include/shadow.h

```diff
@@ -115,11 +117,37 @@ typedef enum ShadowStatus
  */
 #define SHADOW_PREFIX_LENGTH              ( ( uint16_t ) ( sizeof( SHADOW_PREFIX ) - 1U ) )
 
+/**
+ * @ingroup shadow_constants
+ * @brief The root of all un-named "Classic" Shadow MQTT topics
+ *        from here https://docs.aws.amazon.com/iot/latest/developerguide/device-shadow-mqtt.html.
+ */
+#define SHADOW_CLASSIC_ROOT               "/shadow"
```

Comment:
The AWS IoT docs refer to the "Classic" shadow as "unnamed" more often than "classic". I am wondering if we should call this and others UNAMED instead.
```suggestion
#define SHADOW_UNNAMED_ROOT               "/shadow"
```

Reply:
Per our offline discussion, since the AWS Iot Console uses "Classic" rather than "Unnamed", we will stick with that.

![image](https://user-images.githubusercontent.com/3505720/102708317-0f7d2f00-42dd-11eb-9e6e-ce1162acf7a6.png)